services/tenant-ui/frontend/src/store/reservationStore.ts

```diff
@@ -13,7 +13,9 @@ export const useReservationStore = defineStore('reservation', () => {
   // A raw api call without using the interceptors from the acapyApiStore
   // Needed for the open call to reservation at this point
   const api = axios.create({
-    baseURL: config.value.frontend.tenantProxyPath,
+    baseURL: config.value.frontend.showOIDCReservationLogin
+      ? '/'
+      : config.value.frontend.tenantProxyPath,
```

Comment:
Think the goal would be just to proxy through the reservation POST rather than all the ones in the store, see comment about 404)

Reply:
I can create a separate method altogether but thought I’d re use what was there.